hie.yaml

```diff
@@ -0,0 +1,13 @@
+cradle:
+  cabal:
```

Comment:
Can't we leave this to the default behavior by leaving the cradle cabal entry empty?

Reply:
For some project configuration this does not work as expected but it does for out setup so I'll follow your suggestion, thanks!